packages/hack_parallel/hack_parallel.0.1.1/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+maintainer: "rvantonder@gmail.com"
+authors: "Facebook hackers"
+homepage: "https://github.com/rvantonder/hack-parallel"
+bug-reports: "https://github.com/rvantonder/hack-parallel/issues"
+dev-repo: "git+https://github.com/rvantonder/hack-parallel.git"
+license: "MIT"
+build: [
+  [make "libhp.a"]
+  ["dune" "build" "-p" name "-j" jobs "@install"]
+]
+depends: [
+  "core"
+  "ocaml" {>= "4.03.0"}
+  "ppx_deriving"
```

Comment:
Are you sure about not having a lower bound for core and ppx_deriving?

Reply:
I have no idea about what the lower bound is.
And I guess it would be quite slow to search for it.

Reply:
given core install times...

Reply:
I tend to be strict and use what was used for development as constraints, and then if somebody notices or needs it sith dofferent bounds they can be weakened. For example pre v0.9.0 cores are incompatible with post v0.10.0. But the other way around is fine as well, I am going to merge then.